includes/routes/event/edit.php

```diff
@@ -71,6 +73,5 @@ public function handle( int $event_id ): void {
 	 * @throws Exception When date is invalid.
 	 */
 	private static function convertToTimezone( string $date_time, string $time_zone ): string {
```

Comment:
Should this function be removed? It doesn't have a body and doesn't seem to be called anywhere.

Reply:
I removed it, thanks!